CASMINST-4664: Improvements from my experience fresh installing Hermod (#1670)
Cray-HPE/docs-csm: install/prepare_site_init.md

    linux# cp -v ${SITE_INIT}/customizations.yaml ${SITE_INIT}/customizations.yaml.prepassword
    ```

1. Review the configuration to generate these sealed secrets in `customizations.yaml` in the `site-init` directory.

    * `spec.kubernetes.sealed_secrets.cray_reds_credentials`
    * `spec.kubernetes.sealed_secrets.cray_meds_credentials`
    * `spec.kubernetes.sealed_secrets.cray_hms_rts_credentials`

>     The 'cray_reds_credentials' are used by the HMS Discovery Cronjob and the River Endpoint Discovery Service (REDS) for River components. This sealed secret contains the following:

Reviewer: ```suggestion
    The `cray_reds_credentials` are used by the HMS Discovery CronJob and the River Endpoint Discovery Service (REDS) for River components. This sealed secret contains the following:
```

>       * Default Redfish root credentials for air-cooled Node and Router BMCs.

Reviewer: ```suggestion
      * Default Redfish `root` user credentials for air-cooled Node and Router BMCs.
```

>       * Default SNMP credentials configured on leaf-bmc switches. This needs to match the SNMP credentials currently configured on leaf-bmc switches or the credentials will be configured later in the install. The SNMP usernamne, authentication password,

Reviewer: ```suggestion
      * Default SNMP credentials configured on `leaf-bmc` switches. This needs to match the SNMP credentials currently configured on `leaf-bmc` switches or the credentials will be configured later in the install. The SNMP usernamne, authentication password,
```

        and the privacy password need to be provided.

>     The 'cray_meds_credentials' are used by the Mountain Endpoint Discovery Service (MEDS) for the liquid-cooled components in an Olympus (Mountain) cabinet.

Reviewer: ```suggestion
    The `cray_meds_credentials` are used by the Mountain Endpoint Discovery Service (MEDS) for the liquid-cooled components in an Olympus (Mountain) cabinet.
```

>       * The root password needs to match what is currently configured on the CECs for the liquid-cooled cabinets in the system.

Reviewer: ```suggestion
      * The `root` user password needs to match what is currently configured on the CECs for the liquid-cooled cabinets in the system.
```

>     The 'cray_hms_rts_credentials' are used by the Redfish Translation Service (RTS) for any hardware components which are not managed by Redfish, such as a ServerTech PDU in a River Cabinet.

Reviewer: ```suggestion
    The `cray_hms_rts_credentials` are used by the Redfish Translation Service (RTS) for any hardware components which are not managed by Redfish, such as a ServerTech PDU in a River Cabinet.
```

>       * The ServerTech PDU credentials need to match the credentials for the admn user that is currently configured on the PDUs.

Reviewer: ```suggestion
      * The ServerTech PDU credentials need to match the credentials for the `admn` user that is currently configured on the PDUs.
```

>       * The RTS root user credentials are unique to RTS and used only withing the service mesh, and can be set as desired.

Reviewer: ```suggestion
      * The RTS `root` user credentials are unique to RTS and used only within the service mesh; these can be set as desired.
```

    Edit `customizations.yaml` to replace the `Username` and `Password` references in the file so that the values match the **existing settings** of your system hardware components.
    See the `Decrypt Sealed Secrets for Review` section of [Manage Sealed Secrets](../operations/security_and_authentication/Manage_Sealed_Secrets.md#decrypt-sealed-secrets-for-review)

>     if you need to examine credentials from prior installs.

Reviewer: ```suggestion
    if needing to examine credentials from prior installs.
```

        These credentials are used by the REDS and HMS discovery services, targeting River Redfish
        BMC endpoints and management switches

>         > NOTE: For `vault_redfish_defaults`, the only entry used is '{"Cray": {"Username": "root", "Password": "XXXX"}'

Reviewer: ```suggestion
        > NOTE: For `vault_redfish_defaults`, the only entry used is `'{"Cray": {"Username": "root", "Password": "XXXX"}'`
```

                   value: ldaps://dcldap2.us.cray.com
           ```

    1. Configure the `ldapSearchBase` and `localRoleAssignments` settings for
        the `cray-keycloak-users-localize` chart in `customizations.yaml`.

>         There may be one or more groups in LDAP for admins and one or more for users.

Reviewer: ```suggestion
        There may be one or more groups in LDAP for administrators and one or more for users.
```
